packages/devtools_extensions/README.md

```diff
@@ -185,11 +186,13 @@ real DevTools environment. Build your flutter web app and copy the built assets
 
 Use the `build_extension` command from `package:devtools_extensions` to help with this step.
 ```sh
-cd your_extension_web_app &&
-flutter pub get &&
-dart run devtools_extensions build_and_copy \
-  --source=. \
-  --dest=path/to/your_pub_package/extension/devtools 
+cd your_extension_web_app
+```
+```sh
+flutter pub get
+```
+```sh
+dart run devtools_extensions build_and_copy --source=. --dest=path/to/your_pub_package/extension/devtools 
```

Comment:
Maybe `--dest=../foo/extension/devtools` to more closely match the other examples? (They'll likely need to change "foo", but possible the rest would be correct).